mmf/utils/build.py

```diff
@@ -139,6 +139,7 @@ def build_dataloader_and_sampler(
             dataset_instance.dataset_name, dataset_instance.dataset_type
         ),
         num_workers=num_workers,
+        drop_last=False,  # see also MultiDatasetLoader.__len__
```

Comment:
`drop_last=False` is the default so I'm not changing behavior here. I'm just making our choice explicit.